### DISCUSSION TITLE
VSC 如何配置语法提示，以及vue组件引用方式

### DISCUSSION THREAD

**Discussion by charlesooo** [vuejs/vitepress#4446] [Q&A]
用惯了ts，按文档在md文件里写vue，没有vue的语法提示和跳转功能，很难受

该如何配置？

目前是vue组件都单独成文件，但是引用很长：

``` vue
<script setup>
import Home from './Home.vue'
</script>

<Home />
```
可以加个简单的解析方式吗？例如：

```
<./Home.vue />
```

**Comment by xycabcd** [comment]
建议用英语问，回答效率会高很多

**Comment by sharpchen** [comment]
https://github.com/vuejs/vitepress/discussions/4304#discussioncomment-11028831

**Reply by charlesooo** [reply]
tested and didnt work.
tsconfig.json reports "couldnt find any input" and "vitePressExtensions is unkown"
"vue.server.includeLanguages" is unsupported

i think instead of merge different kinds of intellisene in md, it is better to developing in separate ts and vue files. and easily import all to md.

exmaple:

```vue
<script setup>
import Home from './Home.vue'
</script>

<Home />
```

make this shorter, and hook checking on importing

**Reply by sharpchen** [reply]
Not sure about vscode, but works fine in neovim. You may write a full proposal for your idea if you really want that.
